### PR TITLE
Adding missing isView method in ArrayBuffer

### PR DESCRIPTION
**Pull request by hotwinter**

### CODE REVIEW

**Comment by hotwinter** [comment]
@googlebot I signed it!

File: Sources/Fuzzilli/Core/JavaScriptEnvironment.swift [hunk 1]
```diff
@@ -339,7 +340,7 @@ public extension Type {
     static let jsRegExpConstructor = Type.jsFunction([.string] => .jsRegExp)
     
     /// Type of the JavaScript ArrayBuffer constructor builtin.
-    static let jsArrayBufferConstructor = Type.constructor([.integer] => .jsArrayBuffer)
+    static let jsArrayBufferConstructor = .functionAndConstructor([.integer] => .jsArrayBuffer) + .object(ofGroup: "ArrayBufferConstructor", withMethods: ["isView"])
```
Comment (saelo) [inline Sources/Fuzzilli/Core/JavaScriptEnvironment.swift:343]: It's only a constructor, is it not? I.e. 
```
> ArrayBuffer(42)
Uncaught TypeError: Constructor ArrayBuffer requires 'new'
    at ArrayBuffer (<anonymous>)
```

Comment (hotwinter) [reply Sources/Fuzzilli/Core/JavaScriptEnvironment.swift:343]: ah, you are right, sorry, mybad, I was mimicking the way jsArray was doing, so I think I didn't fully understand the type system at this point

**Comment by saelo** [comment]
Thanks!

**Pull request merged**